sbt-ash/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperPlugin.scala

```diff
@@ -0,0 +1,65 @@
+package org.virtuslab.ash
+
+import sbt.Def
+import sbt.Keys._
+import sbt._
+
+object AkkaSerializationHelperPlugin extends AutoPlugin {
+
+  object autoImport extends AkkaSerializationHelperKeys {
+    val baseAshSettings: Seq[Def.Setting[_]] = AkkaSerializationHelperPlugin.baseAshSettings
+  }
+  import autoImport.{baseAshSettings => _, _}
+
+  override def projectSettings: Seq[Def.Setting[_]] = additionalSettings ++ ashSettings
+
+  override def globalSettings: Seq[Def.Setting[_]] = Nil
+
+  lazy val additionalSettings: Seq[Def.Setting[_]] = Seq(
+    libraryDependencies ++= Seq(
+        compilerPlugin(ashDumpEventSchemaCompilerPlugin.value),
+        compilerPlugin(ashRegistrationCheckerCompilerPlugin.value),
+        compilerPlugin(ashSerializabilityCheckerCompilerPlugin.value)),
+    scalacOptions ++= Seq(
+        s"-P:dump-event-schema-plugin:${target.value}",
+        s"-P:registration-checker-plugin:${target.value}"),
+    cleanFiles ++= Seq(
+        target.value / "dump-event-schema-cache",
+        target.value / "registration_checker_plugin_cache.csv"),
+    scalacOptions ++= {
+      val de = (ashDumpEventSchemaCompilerPlugin / ashCompilerPluginEnable).value
+      val dv = (ashDumpEventSchemaCompilerPlugin / ashCompilerPluginVerbose).value
+      val re = (ashRegistrationCheckerCompilerPlugin / ashCompilerPluginEnable).value
+      val rv = (ashRegistrationCheckerCompilerPlugin / ashCompilerPluginVerbose).value
+      val se = (ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginEnable).value
+      val sv = (ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginVerbose).value
+      val l1 = List(de, re, se).map { x =>
+        if (x) "" else "--disable"
+      }
+      val l2 = List(dv, rv, sv).map { x =>
+        if (x) "--verbose" else ""
+      }
+      val plugins = List("dump-event-schema-plugin", "registration-checker-plugin", "serializability-checker-plugin")
+
+      l1.zip(l2).zip(plugins).flatMap { x =>
+        val ((a, b), plugin) = x
+        Seq(a, b).filter(_.nonEmpty).map(x => s"-P:$plugin:$x")
+      }
+    })
+
+  lazy val ashSettings: Seq[Def.Setting[_]] = baseAshSettings
+
+  lazy val baseAshSettings: Seq[Def.Setting[_]] = Seq(
+    ashDumpEventSchemaCompilerPlugin := "org.virtuslab" %% "dump-event-schema-compiler-plugin" % (ashDumpEventSchemaCompilerPlugin / ashCompilerPluginVersion).value,
+    ashRegistrationCheckerCompilerPlugin := "org.virtuslab" %% "registration-checker-compiler-plugin" % (ashRegistrationCheckerCompilerPlugin / ashCompilerPluginVersion).value,
+    ashSerializabilityCheckerCompilerPlugin := "org.virtuslab" %% "serializability-checker-compiler-plugin" % (ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginVersion).value,
+    ashCompilerPluginVersion := "0.2.0",
+    ashCompilerPluginEnable := true,
```

Comment:
Hmmm are all these plugins enabled by default? See the latest commit (_Deliberately comment out some required stuff_) on Hydra `chore/ash-single-sbt-plugin` branch... it compiles without error, as if neither `registration-checker-compiler-plugin` nor `serializability-checker-compiler-plugin` were executed 🤔

Reply:
You forgot to enable the plugin, Let me configure everything.

Reply:
Whoops right 🤦🏻 thanks